Tests/TestingSupport.swift

```diff
@@ -207,6 +207,10 @@ extension Array where Element == CGRect {
 
 extension CGRect: Hashable {
 
+  public func hash(into hasher: inout Hasher) {
+    hasher.combine(hashValue)
```

Comment:
Isn't the proper implementation to use hasher.combine on each of the `CGRect`'s `CGFloat` properties? Then we can delete the `NSStringFromCGRect` stuff.

Reply:
@bryankeller thanks for your comment 🙏 , I updated pull-request 🚀  PTAL  👀